CMakeLists.txt

```diff
@@ -60,6 +60,10 @@ set(CIVETWEB_THREAD_STACK_SIZE 102400 CACHE STRING
 set_property(CACHE CIVETWEB_THREAD_STACK_SIZE PROPERTY VALUE ${CIVETWEB_THREAD_STACK_SIZE})
 message(STATUS "Thread Stack Size - ${CIVETWEB_THREAD_STACK_SIZE}")
 
+# Serve no files from the web server
+option(CIVETWEB_ENABLE_SERVE_FILES "Configures the server to serve static files" ON)
+message(STATUS "Serve files - ${CIVETWEB_ENABLE_SERVE_FILES}")
```

Comment:
It's up to you, modify the commit how you feel fit.

If you want it to be a negative then it should be `CIVETWEB_DISABLE_SERVING_FILES`

To add it to the CI test enviroment you need to enable the flag in the [.travis](https://github.com/bel2125/civetweb/blob/6e292051de2f5864322f8774c523070501e4346c/.travis.yml#L53-L65) and [appveyor](https://github.com/bel2125/civetweb/blob/6e292051de2f5864322f8774c523070501e4346c/appveyor.yml#L91-L103) scripts.